backend/clickhouse/logs.go

```diff
@@ -117,7 +117,7 @@ func (client *Client) LogsKeys(ctx context.Context, projectID int) ([]*modelInpu
 		`
 		SELECT arrayJoin(LogAttributes.keys) as key, count() as cnt
 		FROM logs
-		WHERE ProjectId = ? AND Timestamp < now() AND Timestamp >= (now() - toIntervalDay(30))
```

Comment:
I removed the time constraints because this is functionally equivalent to the [TTL](https://github.com/highlight/highlight/blob/1800d58201aca21876d5fd7d02f802e5e133a579/backend/clickhouse/migrations/000001_create_logs_table.up.sql#L26).